contributing/windows.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Development using Windows
+layout: default
+---
+
+## Setting up the development environment on Windows
+
+This requires Windows 10, 2004 (2020 May Update) or later.

```

Comment:
Maybe include a link to instructions on how to get this version? Because it's not part of regular Windows Update yet, is it? I think there's a link in #15.